apps/registration/migrations/0006_auto_20180125_2313.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2018-01-26 04:13
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
I think you can probably just nuke the database and restart the models from scratch - no need to create unnecessary model things and then remove them in a migration - that'll change once we have a live system